AK/Function.h

```diff
@@ -56,6 +56,14 @@ class Function<Out(In...)> {
     using FunctionType = Out(In...);
     using ReturnType = Out;
 
+#ifdef KERNEL
+    constexpr static auto AccommodateExcessiveAlignmentRequirements = false;
+    constexpr static size_t ExcessiveAlignmentThreshold = alignof(void*);
+#else
+    constexpr static auto AccommodateExcessiveAlignmentRequirements = false;
```

Comment:
When is this ever `true`? What is the point of all the extra code below otherwise?

Reply:
Ah, I meant to flip that to true before pushing here

Reply:
I'll do that in a bit, for now enjoy the fact that there are no uses of Function with alignment>16